examples/stable_diffusion_v2/train_dreambooth.py

```diff
@@ -90,16 +103,10 @@ def parse_args():
         "potentially better generation quality.",
     )
     parser.add_argument(
-        "--train_config",
-        default="configs/train_config.json",
-        type=str,
-        help="Specify the path to the train config file",
-    )
-    parser.add_argument(
-        "--pretrained_model_path", default="", type=str, help="Specify the directory of the pretrained model"
+        "--model_config", default="configs/train_dreambooth_sd_v2.yaml", type=str, help="model config path"
```

Comment:
is there a reason we're not using config/train/*yamls here?

Reply:
We still use config/train/*yamls by train_config argument, which is the first argument in the parser